docker-compose/database/db-changes/users/0005_create_study_user_tables.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE `study_user` (
```

Comment:
I think you need to fill these tables:
![image](https://github.com/user-attachments/assets/01ba92e4-c3a4-44c8-aa14-0bad13bec2b3)

Otherwise, the table will be empty for other users than admins